src/common/components/organisms/FidgetSettingsEditor.tsx

```
@@ -175,19 +181,84 @@ export const FidgetSettingsEditor: React.FC<FidgetSettingsEditorProps> = ({
+
+  // Get current space/tab from zustand
+  const currentSpaceId = useAppStore((state) => state.currentSpace.currentSpaceId);
+  const currentTabName = useAppStore((state) => state.currentSpace.currentTabName);
+
+  // Subscribe directly to zustand for settings - eliminates prop lag
+  const zustandSettings = useFidgetSettings(currentSpaceId, currentTabName, fidgetId);
+
+  // Use zustand settings if available, fall back to props
+  const effectiveSettings = zustandSettings ?? settings;
+
+  const normalizedSettings = useMemo(
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)